experimental/PropertyDDS/packages/property-binder/src/data_binder/dataBinding.ts

```diff
@@ -1106,7 +1108,13 @@ export class DataBinding {
 							}
 							break;
 						case ArrayChangeSetIterator.types.REMOVE:
-							for (i = 0; i < arrayIterator.opDescription.operation![1]; ++i) {
+							for (
+								i = 0;
+								// WARNING: 'operation[1]' is 'string | number | genericArray'.  The cast to 'number'
+								//          preserves the JavaScript coercion behavior, which was permitted prior to TS5.
+								i < (arrayIterator.opDescription.operation![1] as number);
+								++i
+							) {
```

Comment:
This cast preserves the original behavior.

Note that TypeScript 4 permitted implicit coercion to number for things like "5" < 10.
TypeScript 5 has reversed it's stance on this.